examples/Test/Results/002_AL_ORU_R01_NBS_Simplified_0_initial_message.hl7

```diff
@@ -0,0 +1,13 @@
+FHS|^~\&|||0.0.0.0.1|0.0.0.0.1|202106221314-0400
```

Comment:
There's already a group of messages with ID 002. This ID should be unique to given group of messages. It should be 005

Reply:
Fixed the filename now